lib/src/sv_common.c

```diff
@@ -130,6 +130,7 @@ bu_type_to_char(const bu_info_t *bu)
 // SEI UUID types
 const uint8_t kUuidSignedVideo[UUID_LEN] = {
     0x53, 0x69, 0x67, 0x6e, 0x65, 0x64, 0x20, 0x56, 0x69, 0x64, 0x65, 0x6f, 0x2e, 0x2e, 0x2e, 0x30};
+
```

Comment:
nit: Remove this extra line. Then this file is not touched and not part of this commit.